automation/taskcluster/lib/tasks.py

```diff
@@ -332,7 +298,62 @@ def _craft_default_task_definition(
             },
         }
 
-    def craft_signing_task(
+    def craft_master_commit_signing_task(
```

Comment:
Most of the logic here is identical to `craft_nightly_signing_task()`. I would either extract the common bits in smaller functions, parametrize a single function.
Can be done in a follow up

Reply:
Scratch that. I followed your proposal and implemented the same logic in https://github.com/mozilla-mobile/reference-browser/pull/709

Reply:
FWIW, I extracted the common bits into `craft_signing_task(...)` :)